### PR TITLE
shorten search query and concat item name + code for fuzzy search

### PR DESCRIPTION
**Pull request by CedricAngeloSina-on**

### CODE REVIEW

**Review by evad-e — APPROVED** [review]

File: app/api/v1/items/[id]/route.ts [hunk 1]
```diff
@@ -1,47 +1,75 @@
...
-          // Select all
-          // user: true,
-
-          // Exclude column
-          user: {
-            columns: {
-              password: false,
-            }
-          },
+          user: true,
```
Comment (pmvenegas) [inline app/api/v1/items/[id]/route.ts]: Can we specify returned columns, or at least exclude the password?

File: db/drizzle/schema/item.ts [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+
 export const item = pgTable('item', {
   id: serial('id').primaryKey().notNull(),
   code: varchar('code', { length: 10 }).notNull(),
   name: varchar('name', { length: 256 }).notNull(),
   quantity: integer('quantity').notNull(),
   ageing: date('date'),
   description: varchar('description', { length: 256 }),
-  img_path: varchar('image_path', { length: 256 }),
+  img_path: varchar('imagePath', { length: 256 }),
```
Comment (pmvenegas) [inline db/drizzle/schema/item.ts]: Let's use `imagePath` instead of `img_path` to be consistent.

**Review by pmvenegas — CHANGES_REQUESTED** [review]
Some minor changes needed. Thanks!

File: app/api/v1/items/[id]/route.ts (hunk 1 above)
Comment (CedricAngeloSina-on) [reply app/api/v1/items/[id]/route.ts]: we can do that, what specific columns you want to include and exclude

**Comment by CedricAngeloSina-on** [comment]
@pmvenegas pa re-review sir

**Review by pmvenegas — APPROVED** [review]
Thanks, looks great!

**Pull request closed without merging**